install.sh

```diff
@@ -2759,7 +2760,11 @@ parse_command_line_arguments() {
             SINGLE_REPO_INSTALL="yes"
         elif [ "$p" = "--skip-install-into-existing" ]; then
             SKIP_INSTALL_INTO_EXISTING="yes"
+        elif [ "$prev_p" = "--template-dir" ] && [ $p != --* ]; then
```

Comment:
See https://travis-ci.org/rycus86/githooks/jobs/556353067#L525
You could rewrite it like this for example:
```shell
  elif [ "$prev_p" = "--template-dir" ] && (echo "$p" | grep -qE '^\-\-.*'); then
```